Sprint-2/implement/to-pounds.js

```diff
@@ -4,3 +4,18 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+
+function toPounds(penceString){
+    const penceStringWithoutTrailingP = penceString.substring(0, penceString.length - 1);
+    const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+const pounds = paddedPenceNumberString.substring(
+  0,
+  paddedPenceNumberString.length - 2
+);
+const pence = paddedPenceNumberString
+  .substring(paddedPenceNumberString.length - 2)
+  .padEnd(2, "0");
+  return`£${pounds}.${pence}`
```

Comment:
It is never too early to develop the habit to indent the code properly; it makes reading code easier.